src/js/components/Chart/Chart.js

```diff
@@ -19,7 +19,6 @@ const Chart = React.forwardRef(
   (
     {
       a11yTitle,
-      'aria-label': ariaLabel,
```

Comment:
Oops, looks like you removed the aria-label on Chart instead of DataChart

Reply:
Thanks, I noticed chart had `...rest` so I removed and I've included the DataChart stuff on my latest push.